### ISSUE TITLE
TypeError: token.attrGet is not a function

### ISSUE THREAD

**Issue by kadishmal** [valeriangalliat/markdown-it-anchor#24]
```bash
   "TypeError: token.attrGet is not a function",
    "    at /home1/user/61916c91a4e81be8/node_modules/markdown-it-anchor/index.js:79:24",
    "    at Array.forEach (native)",
    "    at Array.<anonymous> (/home1/user/61916c91a4e81be8/node_modules/markdown-it-anchor/index.js:71:8)",
    "    at Core.process (/home1/user/61916c91a4e81be8/node_modules/markdown-it/lib/parser_core.js:51:13)",
    "    at MarkdownIt.parse (/home1/user/61916c91a4e81be8/node_modules/markdown-it/lib/index.js:493:13)",
    "    at MarkdownIt.render (/home1/user/61916c91a4e81be8/node_modules/markdown-it/lib/index.js:513:36)",
    "    at decorate (/home1/user/61916c91a4e81be8/node_modules/aglio-theme-olio/lib/main.js:422:44)",
    "    at Object.exports.render (/home1/user/61916c91a4e81be8/node_modules/aglio-theme-olio/lib/main.js:646:5)",
    "    at /home1/user/61916c91a4e81be8/node_modules/aglio/lib/main.js:133:20"
```

The following works fine. I seem to have installed `v2.5.0` some time ago.

```
`-- grunt-aglio@0.4.1
  `-- aglio@2.2.1
    `-- aglio-theme-olio@1.6.3
      `-- markdown-it-anchor@2.5.0
```

This new version fails.

```
└─┬ grunt-aglio@0.4.1
  └─┬ aglio@2.2.1
    └─┬ aglio-theme-olio@1.6.3
      └── markdown-it-anchor@2.7.0
```

Please add enough test to verify the changes, and if some of them are breaking, always increment the major version, not the minor.

Opened a corresponding issue for `aglio-theme-olio` so that they always use the exact version at https://github.com/danielgtaylor/aglio/issues/319.

**Comment by valeriangalliat** [comment]
Hi, sorry for this, I merged #22 and published it as 2.7.0, looked not breaking to me especially the tests all passed, but I didn't check carefully enough. I'm going to make sure it's not breaking and publish a patch in the following hour.

**Comment by valeriangalliat** [comment]
Hmm it seems to break only with markdown-it < 6.0.0, I didn't check for older versions to consider the change breaking.

I reverted this and published 2.7.1 which should work for you, and published 3.0.0 with the breaking change.

**Comment by kadishmal** [comment]
Great! Thank you @valeriangalliat! You are fast!
